src/url-loader/redirect-resolver.ts

```diff
@@ -37,12 +37,15 @@ export class RedirectResolver extends UrlResolver {
         this.getBaseAndUnresolved(firstUrl, secondUrl);
     const packageRelativeUrl =
         this.brandAsResolved(urlLibResolver(baseUrl, unresolvedUrl));
-    if (packageRelativeUrl === undefined ||
-        !packageRelativeUrl.startsWith(this._redirectFrom)) {
-      return undefined;
+    if (packageRelativeUrl.startsWith(this._redirectFrom)) {
```

Comment:
are there any issues with missing/extra leading/trailing slashes here, by doing startsWith? (would have been an issue anyway, just curious)

Reply:
There's no extra-slash problem I can think of, but I think there's a potential that a redirect resolver that didn't enforce a trailing slash on the redirect-from could wind up resolving urls in a sibling directory unintentionally like `/folder` redirecting to `/folder3` could redirect `/folder2/page.html` to `/folder32/page.html`.  I think that class of problem should be looked at separately.  Added follow-up https://github.com/Polymer/polymer-analyzer/issues/844